cmd/stat.go

```diff
@@ -112,6 +113,14 @@ func parseStat(c *clientContent) statMessage {
 	return content
 }
 
+// Return standardized URL to be used to compare later.
+func getStandardizedURL(targetURL string) string {
+	if runtime.GOOS == "windows" {
+		targetURL = strings.Replace(targetURL, "/", "\\", -1)
```

Comment:
We should use filepath.ToSlash() as needed here not strings.Replace()

Reply:
Used filepath.FromSlash() instead of Strings.Replace